fms/models/gpt_bigcode.py

```diff
@@ -15,17 +15,15 @@
 class GPTBigCodeConfig(ModelConfig):
     src_vocab_size: int = 49157  # This param default is based on https://huggingface.co/bigcode/gpt_bigcode-santacoder
     emb_dim: int = 2048  # This param default is based on https://huggingface.co/bigcode/gpt_bigcode-santacoder
-    emb_kq: Optional[int] = None
-    emb_v: Optional[int] = None
     nheads: int = 12
-    kvheads: int = 1
     nlayers: int = 12
     pad_id: int = 0
     max_pos: int = 512
     hidden_grow_factor: float = 4.0
     activation_fn: str = "gelu-tanh"
     p_dropout: float = 0.0
     emb_dropout: float = 0.0
+    multiquery_attn: bool = True
```

Comment:
is there a non-MQA variant of this model that we plan to support?

Reply:
Our models support both, so we should keep the boolean for now